Sming/Arch/Host/Platform/StationImpl.cpp

```diff
@@ -238,45 +243,33 @@ String StationImpl::getHostname() const
 
 IPAddress StationImpl::getIP() const
 {
-	netif* nif = netif_default;
-
-	return nif ? nif->ip_addr.addr : IPADDR_NONE;
+	return ipaddr;
 }
 
-MACAddress StationImpl::getMacAddr() const
+MACAddress StationImpl::getMacAddress() const
 {
 	netif* nif = netif_default;
 
 	if(nif == nullptr) {
 		return MACADDR_NONE;
 	} else {
-		return nif->hwaddr;
+		return MACAddress(nif->hwaddr);
 	}
 }
 
 IPAddress StationImpl::getNetworkBroadcast() const
```

Comment:
Rename `IPAddress` to `IpAddress`. Deprecate IPAddress and add a note in the migration documents about it.